userreg/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.3.10
```

Comment:
this version is pretty old, I suggest to use at least `1.5.8`

Reply:
I suppose that this is an old typo.
This `build.properties` is not involved in any building process.

Reply:
removed auto-generation of build.properties file for userreg module